react/src/pages/analysis/Analysis.tsx

```diff
@@ -65,7 +60,14 @@ const useStyles = makeStyles(theme => ({
 
 }));
 
-interface AnalysisRun {
+export enum PipelineStatus {
+    PENDING = "Pending",
+    RUNNING = "Running",
+    COMPLETED = "Completed",
+    ERROR = "Error",
+}
```

Comment:
This is a good change so this can get type-checked too instead of just being a string.